.clang-format

```diff
@@ -306,22 +306,22 @@ IncludeCategories:
   - Regex:           '^"pyhelpers/'
```

Comment:
@AndrewShkrob, this patch looks good at glance. However, there are some concerns. Mail.Ru had a lot of time and resources to do such refactoring when MapsWithMe was rebranded to MapsMe. And they didn't. Why? I don't know. I am pretty sure that they tried multiple times. I have heard a belief from some folks that changing Java package name can somehow affect Google Play reviews. It sounds like fairy tales, but who knows? :-) Have you tried such renaming in the past on a production app?

One more issue we can address is to configure `.git-blame-ignore-revs` to improve `git blame`:

https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view

Reply:
Unfortunately, I don't have such experience in renaming production apps, but I found some limitations connected with google play [here](https://android-developers.googleblog.com/2011/06/things-that-cannot-change.html)

> Once you publish your application under its manifest package name, this is the unique identity of the application forever more. Switching to a different name results in an entirely new application, one that can’t be installed as an update to the existing application.

> Just as important as the manifest package name is the certificate that application is signed with. The signing certificate represents the author of the application. If you change the certificate an application is signed with, it is now a different application because it comes from a different author. This different application can’t be uploaded to Market as an update to the original application, nor can it be installed onto a device as an update.

> The exact behavior the user sees when installing an application that has changed in one of these two ways is different:

> If the manifest package name has changed, the new application will be installed alongside the old application, so they both co-exist on the user’s device at the same time.

> If the signing certificate changes, trying to install the new application on to the device will fail until the old version is uninstalled.

I don't know if it is our case, because I don't see `package` attribute defined in `<manifest>`. Maybe it is hidden and available only on your side? 

I can configure `.git-blame-ignore-revs`, no problem. But, I think, at first we should decide if this pull request would be accepted.

Anyway, sooner or later, such patch has to be applied 😊

Reply:
>   Once you publish your application under its manifest package name, this is the unique identity of the application forever more. Switching to a different name results in an entirely new application, one that can’t be installed as an update to the existing application.

We don't change the package name in manifest. It should be fine.

> I can configure .git-blame-ignore-revs, no problem. But, I think, at first we should decide if this pull request would be accepted.

Let's try this `git-blame-ignore-revs` and see how it works.